src/scatter_buildblock/ScatterEstimation.cxx

```diff
@@ -77,6 +77,7 @@ ScatterEstimation::set_defaults()
   this->override_scanner_template = true;
   this->override_density_image = true;
   this->downsample_scanner_bool = true;
+  this->downsampled_detectors_per_ring = 0;
```

Comment:
normally default variables like this should be -1, such that we know it's invalid. The code indeed checks for `<= 0`, so it's fine, but I'd prefer -1...